### ISSUE TITLE
Upload requires all blobbers to work properly

### ISSUE THREAD

**Issue by fabioDMFerreira** [0chain/blobber#808]
n data + 1 parity blobbers should be enough to allow a client to upload.

**Steps**
1. Launch network with 3 miners, 1 sharder and 4 blobbers.
2. Adapt one of the blobbers to not respond to the upload request.
3. Create an allocation, stake tokens and upload a file.

**Expected**
File to be uploaded.

**Actual**
The upload fails with the next errors
```
0box-sdk        [ERROR]  2022/08/22 16:26:25.190068 chunked_upload_blobber.go:85: http://198.18.0.91:5051/ Upload response parse error: unexpected end of JSON input
0box-sdk        [ERROR]  2022/08/22 16:26:25.191036 chunked_upload_blobber.go:67: Upload : Post "http://198.18.0.94:5054/v1/file/upload/233762504de8c165b4f0fe6d813e5f02f96c3f99d7b5165251cafd42f1b34d31": context canceled
0box-sdk        [ERROR]  2022/08/22 16:26:25.191041 chunked_upload_blobber.go:67: Upload : Post "http://198.18.0.93:5053/v1/file/upload/233762504de8c165b4f0fe6d813e5f02f96c3f99d7b5165251cafd42f1b34d31": context canceled
0box-sdk        [ERROR]  2022/08/22 16:26:25.191107 chunked_upload_blobber.go:67: Upload : Post "http://198.18.0.92:5052/v1/file/upload/233762504de8c165b4f0fe6d813e5f02f96c3f99d7b5165251cafd42f1b34d31": context canceled
```

**More info**
The command used to create an allocation was
```
./zboxcli/zbox --wallet testing.json newallocation \
    --read_price 0.001-10 --write_price 0.01-10 --size 104857600 --lock 4 \
    --data 2 --parity 2 --expire 48h
```

**Comment by fabioDMFerreira** [comment]
Worth to check after the fix of this issue whether #791 and #780 are resolved.

**Comment by lpoli** [comment]
Same issue raised [here](https://github.com/0chain/gosdk/issues/545)
